yt_dlp/extractor/voot.py

```diff
@@ -34,17 +44,13 @@ class VootIE(InfoExtractor):
             'skip_download': True,
         },
         'expected_warnings': ['Failed to download m3u8 information'],
-    }, {
-        'url': 'https://www.voot.com/kids/characters/mighty-cat-masked-niyander-e-/400478/school-bag-disappears/440925',
-        'only_matching': True,
```

Comment:
even though this URL no longer works, we have no reason to intentionally break it